tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -435,10 +435,10 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                               help=argparse.SUPPRESS, action=DeprecatedStoreTrue, default=False)
     common_group.add_argument("--use_new_frontend",
                               help="Use new frontend API for model processing",
-                              action='store_true', default=False)
+                              action='store_true', default=argparse.SUPPRESS)
     common_group.add_argument("--use_legacy_frontend",
                               help="Use legacy API for model processing",
-                              action='store_true', default=False)
+                              action='store_true', default=argparse.SUPPRESS)
```

Comment:
please do not suppress these options since we use them for new TF frontend (validation, etc.)

Reply:
done